### PR TITLE
Adding wireguard info

### PR DESCRIPTION
**Pull request by monroeclinton**
Adding wireguard info

### CODE REVIEW

File: getting-started/troubleshooting.html.md [hunk 1]
```diff
@@ -142,4 +142,29 @@ Use these two commands to see environment variables.
...
 flyctl config env
-```
\ No newline at end of file
+```
+
+## _Troubles with WireGuard_
+
+To check if you are connected to the private network you can use the command to get a list of your private networking apps in the same organization.
+```
+dig +short _apps.internal
```
Comment (mrkurt) [inline getting-started/troubleshooting.html.md:151]: Dig probably needs the DNS server specified. Something like `dig +short _apps.internal @fdaa:0:1:3`. This varies per peer, it's included in the wireguard config we generate.

File: getting-started/troubleshooting.html.md [hunk 2]
```diff
@@ -142,4 +142,29 @@ Use these two commands to see environment variables.
...
+```
+
+If this does not return anything and you have existing apps, then you may not be connected to the private network. Check your WireGuard configuration or recreate it.
+
+### IPV6 applications
+
+Fly's [private network](/docs/reference/privatenetwork/) is connected together using WireGuard tunnels and IPV6. If you are having trouble resolving one of your applications, make sure you are connecting to it with a method that is compaitable with IPV6. For example, to ping you may have to use:
```
Comment (mrkurt) [inline getting-started/troubleshooting.html.md:158]: Small typo on "compatible".

**Review by mrkurt — CHANGES_REQUESTED** [review]

File: getting-started/troubleshooting.html.md (hunk 1 above)
Comment (tqbf) [reply getting-started/troubleshooting.html.md:151]: There's language about this in the documentation here: 

https://fly.io/docs/reference/privatenetwork/

File: getting-started/troubleshooting.html.md [hunk 3]
```diff
@@ -142,4 +142,29 @@ Use these two commands to see environment variables.
 ```
 flyctl secrets list
 flyctl config env
-```
\ No newline at end of file
+```
+
+## _Troubles with WireGuard_
```
Comment (10thfloor) [inline getting-started/troubleshooting.html.md:147]: To get Wireguard connection working on OSx I had to add the DNS server manually (fdaa:0:2e91::3), using Network settings GUI. 

I also had to disable other non-wireguard VPNs

**Comment by jsierles** [comment]
We have a new troubleshooting guide on the way, and will incorporate the issues raised here. Thanks!

**Pull request closed without merging**